mods/d2k/rules/ai.yaml

```diff
@@ -48,14 +48,12 @@ Player:
 			upgrade.hightech: 0.1%
 		UnitsToBuild:
 			carryall: 1%
-			light_inf: 6%
-			trooper: 5%
-			medic: 1%
-			fremen: 0.5%
-			sardaukar: 1.5%
-			saboteur: 0.5%
+			light_inf: 65%
+			trooper: 40%
+			thumper: 1%
```

Comment:
I'd leave those out, the AI doesn't know what to do with them (ie., how and where to deploy them).